specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Line 307:
```
  election_outcome: TYPE = set[contest_outcome]
```
I would change that to `election_outcomes`, since it is not a summary, 
and the contest outcomes are more or less independent.